airflow-core/src/airflow/ui/src/components/EditableMarkdownNotes.tsx

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import EditableMarkdownArea from "./EditableMarkdownArea";
+import { Accordion } from "./ui/Accordion";
+
+const EditableMarkdownNotes = ({
```

Comment:
I don't think the name of the component is super descriptive. (vs `EditableMarkdownArea`)

Because it leaves us with `Area` vs `Notes`, impossible to guess who is collapsible, who's not, what's the difference.